api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupTXNEW.java

```diff
@@ -95,30 +92,26 @@ public String getDescription() {
 	@Override
 	public ReportDefinition constructReportDefinition() {
 		
-		CohortIndicatorDataSetDefinition dataSetDefinition = new CohortIndicatorDataSetDefinition();
-		dataSetDefinition.setName("TX_NEW Data Set");
-		dataSetDefinition.addParameter(ReportingConstants.START_DATE_PARAMETER);
-		dataSetDefinition.addParameter(ReportingConstants.END_DATE_PARAMETER);
+		CohortIndicatorDataSetDefinition dataSetDefinition = CohortIndicatorDataSets.newCohortIndicatorDataSetDefinition("TX_NEW Data Set", getParameters());
```

Comment:
@rubailly I was actually thinking that we extract this report definition into the library folder as the `TxNewDataSet` so that we can reuse it in the MER_Quarterly report when we get there. What do you think?